components/tools/OmeroFS/src/fsMonitor.py

```diff
@@ -57,12 +54,22 @@ class AbstractMonitor(object):
     """
 
     def __init__(self, eventTypes, pathMode, pathString, whitelist, blacklist,
-                 ignoreSysFiles, ignoreDirEvents, proxy, monitorId):
+                 ignoreSysFiles, ignoreDirEvents, platformCheck, proxy,
+                 monitorId):
         """
             Initialise Monitor.
 
         """
         self.log = logging.getLogger("fsclient." + __name__)
+
+        # Now try to import the correct MonitorServer package
+        import fsUtil
+        try:
+            PlatformMonitor = __import__(
+                fsUtil.monitorPackage(platformCheck))
+        except:
+            raise
```

Comment:
Probably, yes. In its previous place it'd been there a long time! Maybe I once intended there to be some logging there!